test/e2e/TestEnv.ts

```diff
@@ -13,13 +13,16 @@ export type TestEnvType = {
     dfspa: string,
```

Comment:
I imagine we will likely want to refactor and make this config more generic and configurable, but this is fine for now.

Reply:
Okay